src/modules/add-site/hooks/use-stepper.ts

```diff
@@ -66,6 +68,18 @@ export function useStepper( config?: StepperConfig ): UseStepper {
 			{ id: 'site-details', label: __( 'Site name & details' ), path: '/pullRemote/create' },
 		];
 
+		const blueprintDeeplinkSteps: StepperStep[] = [
+			{ id: 'blueprint-selected', label: __( 'Blueprint selected' ), path: '/blueprintDeeplink' },
```

Comment:
```suggestion
			{ id: 'blueprint-selected', label: __( 'Blueprint details' ), path: '/blueprintDeeplink' },
```
In this case user didn't select blueprint so showing "Blueprint selected" doesn't look right. I would suggest to change it to "Blueprint details" or something else to reflect text that is more closer to user action.

Reply:
Good point! I have changed the step label to `Blueprint details` in https://github.com/Automattic/studio/pull/2130/commits/34aa07c89a562daefc1f7821957f52e54bf62a22.